deploy/charts/emqx-operator/templates/controller-manager-rbac.yaml

```diff
@@ -271,32 +271,6 @@ rules:
   - get
   - patch
   - update
-- apiGroups:
-  - apps.emqx.io
-  resources:
-  - rebalances
-  verbs:
-  - create
-  - delete
-  - get
-  - list
-  - patch
-  - update
-  - watch
-- apiGroups:
-  - apps.emqx.io
-  resources:
-  - rebalances/finalizers
-  verbs:
-  - update
-- apiGroups:
-  - apps.emqx.io
-  resources:
-  - rebalances/status
-  verbs:
-  - get
-  - patch
-  - update
```

Comment:
Why delete this ?

Reply:
Will add back when coding controller，otherwise, ci failed